perf-test-jenkins/Main.groovy

```diff
@@ -53,56 +66,52 @@ println("REPORT_BASE_DIR: ${REPORT_BASE_DIR}")
 @Field final File SUMMARIES_OUTPUT_DIR = new File("${args[1]}/test_output_summaries")
 
 // Directory where to strore exported csv files and rule provider reports
-@Field final File SUMMARY_OUTPUT_DIR = getCurrentRuleSummaryDirectory()
+@Field final File SUMMARY_OUTPUT_DIR = new File(getCurrentRuleSummaryDirectory(), TEST_APP_NAME)
 SUMMARY_OUTPUT_DIR.mkdirs()
 println("SUMMARY_OUTPUT_DIR: ${SUMMARY_OUTPUT_DIR}")
+if (SUMMARY_OUTPUT_DIR.list().length > 0) {
+    println("Error: summary output dir should be empty")
+    System.exit(1)
```

Comment:
I guess non-empty dir would mean that previous clean-up went wrong, in which case, perhaps cleaning the dir and continuing would be more robust.